pkg/sql/lease_internal_test.go

```diff
@@ -284,24 +287,28 @@ CREATE TABLE t.test (k CHAR PRIMARY KEY, v CHAR);
 		t.Fatal(err)
 	}
 
-	tableDesc := sqlbase.GetTableDescriptor(kvDB, "t", "test")
+	tableDesc := sqlbase.GetTableDescriptor(kvDB, "t", tableName)
 
 	// Check the assumptions this tests makes: that there is a cache entry
 	// (with a valid lease).
-	lease := leaseManager.tableNames.get(tableDesc.ParentID, "test", s.Clock())
-	if lease == nil {
-		t.Fatalf("no name cache entry")
-	}
-	if err := leaseManager.Release(lease); err != nil {
-		t.Fatal(err)
+	if lease := leaseManager.tableNames.get(tableDesc.ParentID, tableName, s.Clock()); lease == nil {
+		t.Fatalf("name cache has no unexpired entry for (%d, %s)", tableDesc.ParentID, tableName)
+	} else {
+		if err := leaseManager.Release(lease); err != nil {
+			t.Fatal(err)
+		}
+
+		// Manually expire the lease. This is pretty gross, but such is life.
+		leaseManager.tableNames.mu.Lock()
+		lease.expiration = parser.DTimestamp{
+			Time: s.Clock().Now().GoTime().Add(-time.Millisecond),
+		}
+		leaseManager.tableNames.mu.Unlock()
```

Comment:
can we move this code to where the lease manager is? I can see it being reused elsewhere